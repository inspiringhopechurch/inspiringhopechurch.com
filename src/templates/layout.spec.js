import React from "react";
import { shallow } from "enzyme";
import Layout from "./layout";

describe("Layout", () => {
  let myLayout = undefined;

  beforeEach(() => {
    myLayout = shallow(<Layout />);
  });

  it("should render without crashing", () => {
    expect(myLayout).toMatchSnapshot();
  });
});
