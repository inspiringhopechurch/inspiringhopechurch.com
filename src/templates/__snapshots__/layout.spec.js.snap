// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render without crashing 1`] = `
<Fragment>
  <SEO
    article={false}
    banner={null}
    desc={null}
    pathname={null}
    title={null}
  />
  <Header />
  <Footer
    copyrightOwner="Inspiring Hope Church"
  />
</Fragment>
`;
